src/pages/user/authority/NewAuthorityCard.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useDispatch, useSelector } from 'react-redux';
+import styled from 'styled-components';
+
+import { Button, Radio } from '@material-ui/core';
+
+import { mockSingleAuthorityResponseWithFeide } from '../../../api/mock-interceptor';
+import Progress from '../../../components/Progress';
+import { setAuthorityData } from '../../../redux/actions/userActions';
+import { RootStore } from '../../../redux/reducers/rootReducer';
+
+const StyledBoxContent = styled.div`
+  display: grid;
+  grid-template-areas:
+    'authority authority authority'
+    'description description description'
+    '. . create-button';
+  grid-template-columns: 1fr 3fr 3fr;
+  background-color: ${({ theme }) => theme.palette.box.main};
+  padding: 1rem;
+  align-items: center;
+  height: 16rem;
+`;
+
+const StyledAuthority = styled.div`
+  grid-area: authority;
+`;
+
+const StyledDescription = styled.div`
+  grid-area: description;
+  margin-left: 0.7rem;
+`;
+
+const StyledButton = styled(Button)`
+  grid-area: create-button;
+  margin-top: 2rem;
+`;
+
+const StyledProgressContainer = styled.div`
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  width: 36rem;
+  height: 16rem;
+  padding: 2rem;
+`;
+
+const NewAuthorityCard: React.FC = () => {
+  const dispatch = useDispatch();
+  const user = useSelector((store: RootStore) => store.user);
+  const [loading, setLoading] = useState(false);
+  const { t } = useTranslation('profile');
+
+  const handleCreateAuthority = async () => {
+    // NOT YET IMPLEMENTED IN BACKEND
+    // setLoading(true);
+    // const authority = await createAuthority(user.name);
+    // if (authority) {
+    //   dispatch(setAuthorityData(authority));
+    //   setLoading(false);
+    // }
+    // if (USE_MOCK_DATA) {
+    setTimeout(() => {
+      dispatch(setAuthorityData(mockSingleAuthorityResponseWithFeide));
+      setLoading(false);
+    }, [2000]);
```

Comment:
timout bare for å få det mest mulig autentisk for tester, eller er det noe mer?

Reply:
fordi det kan ta mellom 0,5-4 sekund å få svar fra BARE (og for at spinneren skal vises litt)